### PR TITLE
ola: update to 0.10.5, drop obsolete patches

### PR DESCRIPTION
**Pull request by bk138**
-------------------------------

Maintainer: Christian Beier <dontmind@freeshell.org>
Compile tested: ar71xx, OpenWRT 15.05
Run tested: ar71xx, OpenWRT 15.05, Basic DMX receive, cmdline util tests

Description: This updates OLA to the recently released version 0.10.5 and drops some patches that were incorporated upstream.

### CODE REVIEW

**Comment by bk138** [comment]
argh, seems this is blocked by https://github.com/OpenLightingProject/ola/issues/1192. shall I delete this PR then or update it when upstream happens to fix mentioned issue?

File: net/ola/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
-# Copyright (C) 2006-2015 OpenWrt.org
-# Copyright (C) 2015 Christian Beier <dontmind@freeshell.org>
+# Copyright (C) 2006-2017 OpenWrt.org
```
Comment (champtar) [inline net/ola/Makefile]: small detail, don't update OpenWrt.org copyright as you are not affiliated with it (or are you?)

**Comment by champtar** [comment]
Hi @bk138,
I prefer to close it to keep the backlog of PR smaller,
and you can reopen it / update it when everything is working

**Comment by bk138** [comment]
fine with me

**Comment by peternewman** [comment]
@bk138 there is seemingly a fix from @cmuellner here https://github.com/OpenLightingProject/ola/pull/1336 . Do you want to try that as a local patch your end, which is probably the quickest way to get you up and running again, and then ideally give the patched OLA a bit of a hammering to see if it's unexpectedly broken anything?

**Comment by bk138** [comment]
@peternewman I'll have a look ASAP.

**Comment by bk138** [comment]
@peternewman builds fine now, will do some runtime testing soonish as well.

**Comment by bk138** [comment]
@peternewman runtime tests succeeded as well, I can send ArtNet from a OpenWRT box to a laptop  and vice versa.

**Pull request closed without merging**